src/components/results/sensitivity-analysis/sensitivity-analysis-result.js

```diff
@@ -46,6 +53,10 @@ const SensitivityAnalysisResult = ({
     sortProps,
     filterProps,
     isLoading,
+    handleCsvHeadersChange,
+    exportResultAsCsv,
```

Comment:
In my opinion, it is more conventional with

onCsvHeadersChange: handleCsvHeadersChange, 
onExportResultAsCsv: handleExportResultAsCsv,

And add 4 new props to propTypes